action.yml

```diff
@@ -66,10 +66,13 @@ runs:
       shell: bash
       run: |
         if [ "${{ inputs.mesa3d-release }}" == "latest" ]; then
+          echo "Fetching latest Mesa3D release version..."
           export MESA3D_VERSION=$(curl -s https://api.github.com/repos/pal1000/mesa-dist-win/releases/latest | grep -o '"tag_name": ".*"' | sed 's/"tag_name": "\(.*\)"/\1/')
         else
+          echo "Using specified Mesa3D release version..."
           export MESA3D_VERSION=${{ inputs.mesa3d-release }}
         fi
+        echo "Mesa3D version to be used: ${MESA3D_VERSION}"
```

Comment:
The nice thing about the `| tee -a` below is that you don't need this bit. The `MESA3D_VERSION=` that you see in the changelog is already there, e.g.:

https://github.com/pyvista/setup-headless-display-action/actions/runs/12121147123/job/33791450794#step:3:18

Reply:
Ah you are right @larsoner - good catch